packages/cisco_ise/data_stream/log/elasticsearch/ingest_pipeline/pipeline_passed_authentications.yml

```
@@ -466,27 +481,35 @@ processors:
       field: cisco_ise.log.log_details.UserName
       ignore_missing: true
   - append:
+      tag: append_user_name_1
       field: user.name
       value: '{{{cisco_ise.log.log_details.User-Name}}}'
-      if: ctx.cisco_ise?.log?.log_details['User-Name'] != null
+      if: >-
+        ctx.cisci_ise?.log != null && ctx.cisco_ise?.log?.log_details != null &&
+        ctx.cisco_ise?.log?.log_details['User-Name'] != null
       allow_duplicates: false
       ignore_failure: true
   - append:
+      tag: append_related_user_1
       field: related.user
       value: '{{{cisco_ise.log.log_details.lUser-Name}}}'
```

Looks like there's a typo in this line:
```suggestion
      value: '{{{cisco_ise.log.log_details.User-Name}}}'
```

---

thanks, good catch